comp/core/flare/helpers/send_flare.go

```diff
@@ -254,13 +272,65 @@ func SendTo(cfg pkgconfigmodel.Reader, archivePath, caseID, email, apiKey, url s
 		return "", err
 	}
 
-	r, err := readAndPostFlareFile(archivePath, caseID, email, hostname, url, source, client, apiKey)
-	if err != nil {
-		return "", err
+	// Retry logic for the actual flare file posting
+	var lastErr error
+	var maxTries = 3
+	var baseDelay = 100 * time.Millisecond
+
+	for attempt := 1; attempt <= maxTries; attempt++ {
```

Comment:
We don't need 2 variables to track retries.

```suggestion
	var lastErr error
	var baseDelay = 100 * time.Millisecond

	for attempt := 3; attempt > 0; attempt-- {
```